schema/manifest_backwards_compatibility_test.go

```diff
@@ -137,12 +137,12 @@ func TestBackwardsCompatibilityManifest(t *testing.T) {
 		// "Accept: application/vnd.docker.distribution.manifest.v2+json" \
 		// https://registry-1.docker.io/v2/library/docker/manifests/sha256:888206c77cd2811ec47e752ba291e5b7734e3ef137dfd222daadaca39a9f17bc
 		{
-			digest: "sha256:888206c77cd2811ec47e752ba291e5b7734e3ef137dfd222daadaca39a9f17bc",
+			digest: "sha256:d901d5244c94db8151415bf3a1d3a0b10e7468e4997786b01c287cbcbaeca446",
 			manifest: `{
    "schemaVersion": 2,
    "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
    "config": {
-      "mediaType": "application/octet-stream",
+      "mediaType": "application/vnd.docker.container.image.v1+json",
```

Comment:
Can we do this?  The octet-stream type is from #145, where @philips made it sound like this manifest had been caught in the wild.  If that's the case, we shouldn't be messing with the source.